src/cardpage.js

```diff
@@ -70,9 +70,9 @@ class CardPage extends Component {
             </Col>
             <Col className="breakdown" xs="12" sm="8">
               <p>
-                Played in
+                Played in &nbsp;
```

Comment:
I am not sure on the best practice here, but I think instead of adding `&nbsp;` we'd want `{' '}`... I am up to discuss this though as like I said, I am not sure the actual best practices here.

Reply:
Thanks for the idea! I hadn't seen that before.  

I added these in, and the `prettier` plugin complained about the first one, so i took its suggestion and reformatted a bit.

Thanks again!

Reply:
yea, it's a weird one for sure. This gives a little bit more information as to my thought process...

https://stackoverflow.com/questions/40264084/best-practice-when-adding-whitespace-in-jsx

Reply:
I definitely prefer using {' '} for white space in JSX

Reply:
Done, thanks!  (I forgot to push changes earlier, lol)